clipper_admin/clipper_manager.py

```diff
@@ -1267,3 +1267,73 @@ def _put_container_on_host(self, container_name):
             warn("Could not find %s, please try with a valid "
                  "container docker image")
             return False
+
+    def deploy_R_model(self,
+                       name,
+                       version,
+                       model_data,
+                       container_name,
+                       input_type,
+                       labels=DEFAULT_LABEL,
+                       num_containers=1):
+        """Registers a model with Clipper and deploys instances of it in containers.
+        Parameters
+        ----------
+        name : str
+            The name to assign this model.
+        version : int
+            The version to assign this model.
+        model_data : 
+            The trained model to add to Clipper.The type has to be rpy2.robjects.vectors.ListVector,
+            this is how python's rpy2 encapsulates any given R model.This model will be loaded
+            into the Clipper model container and provided as an argument to the
+            predict function each time it is called. 
+        container_name : str
+            The Docker container image to use to run this model container.
+        input_type : str
+            "strings" (from which model specific dataframes can be derived for carrying out predictions). 
+        labels : list of str, optional
+            A set of strings annotating the model 
+        num_containers : int, optional
+            The number of replicas of the model to create. More replicas can be
+            created later as well. Defaults to 1.
+        """
+
+        # importing some R specific dependencies
+        import rpy2.robjects as ro
+        from rpy2.robjects.packages import importr
+        base = importr('base')
+
```

Comment:
Because you removed the container_name and input_type arguments from the method, set the variables here:

```py
container_name = "clipper/r_python_container"
input_type = "strings"
```

Also make sure to update the examples and tests in the rest of the PR.